javaslang-benchmark/src/test/java/javaslang/benchmark/collection/ListBenchmark.java

```diff
@@ -1,47 +1,28 @@
 package javaslang.benchmark.collection;
 
-import javaslang.benchmark.JmhRunner;
-import org.openjdk.jmh.annotations.Benchmark;
-import org.openjdk.jmh.annotations.Level;
-import org.openjdk.jmh.annotations.Param;
-import org.openjdk.jmh.annotations.Scope;
-import org.openjdk.jmh.annotations.Setup;
-import org.openjdk.jmh.annotations.State;
-import org.openjdk.jmh.annotations.TearDown;
-
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.Objects;
-import java.util.Random;
+import org.openjdk.jmh.annotations.*;
+import scala.compat.java8.JFunction;
 
-public class ListBenchmark {
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static javaslang.benchmark.JmhRunner.*;
 
+public class ListBenchmark {
     public static void main(String... args) { /* main is more reliable than a test */
-        JmhRunner.run(ListBenchmark.class);
+        run(ListBenchmark.class);
     }
 
     @State(Scope.Benchmark)
     public static class Base {
-        @Param({ "10", "100", "1000", "10000"})
+        @Param({ "10", "100", "1000" })
```

Comment:
for linear data structures `1000` should be enough (`fjava` is too slow for larger sets)